cli/README.md

```diff
@@ -57,6 +57,95 @@ yo mern:component Sidebar --stateless
 
 Produces `app/components/Sidebar.js`.
 
```

Comment:
Super minor: What do you think @matiasbeckerle to add here in this readme a short note that the `yo` command has to be run in the root of the project, and not in the `/cli` folder?